src/components/page-intro.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { formatLargeNumber } from '../utils';
+import '../styles/components/page-intro.scss';
+import ChevronDown from '../svg/chevron-down.svg';
+import ChevronUp from '../svg/chevron-up.svg';
+
+class PageIntro extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showContent: false,
+    };
+  }
+
+  render() {
+    const {
+      title, resultsCount, children, links,
+    } = this.props;
+    const { showContent } = this.state;
+    return (
+      <div className="page-intro">
+        <h2>
+          <button
+            type="button"
+            onClick={() => this.setState({ showContent: !showContent })}
+            className="dropdown"
+          >
+            {showContent ? <ChevronUp /> : <ChevronDown />}
+            {' '}
+            {title}
+          </button>
+          {resultsCount > 0 && (
+          <small>
+            {' '}
+            {formatLargeNumber(resultsCount)}
+            {' '}
+results
+          </small>
+          )}
+        </h2>
+
+        <div className="intro-content" style={{ display: showContent ? 'block' : 'none' }}>
```

Comment:
Maybe instead of inline-styling, we can have a BEM Modifier CSS class instead e.g. `intro-content--display-content`.

Reply:
Yup